### ISSUE TITLE
declarations to 3.9.0 respond with "Installing service discovery components..."

### ISSUE THREAD

**Issue by kreynoldsf5** [F5Networks/f5-appsvcs-extension#81]
##### Do you already have an issue opened with F5 support?
No.

##### ISSUE TYPE

 - Bug Report

##### AS3 BUILD/ VERSION

```json
{
  "version": "3.9.0",
  "release": "3",
  "schemaCurrent": "3.9.0",
  "schemaMinimum": "3.0.0"
}
```

##### BIGIP VERSION

```
Sys::Version
Main Package
  Product     BIG-IP
  Version     13.1.1.4
  Build       0.0.4
  Edition     Point Release 4
  Date        Wed Jan 16 18:04:36 PST 2019
```

##### SUMMARY

```
BIG-IP does not have AS3 installed. All declaration through f5devcentral/f5-as3-container:latest (3.9.0) yield the following result:
```
```json
{
  "results": "Installing service discovery components. The results of your request may be retrieved by sending a GET request to selfLink provided.",
  "id": "9212c0b0-e598-46a5-8a7e-7a43674a88d5",
  "selfLink": "https://10.10.0.138/mgmt/shared/appsvcs/task/9212c0b0-e598-46a5-8a7e-7a43674a88d5"
}
```
```
This happens with *any* declaration regardless of whether addressDiscovery is used. I've been testing with the standard Sample_01 provided. I will also attempt to repro directly against a BIG-IP with AS3 installed. Note the selfLink returned is the BIG-IP and not the container.
If I update my pipeline to use f5devcentral/f5-as3-container:3.7.0 (everything else the same) the declaration works as expected.
```

##### STEPS TO REPRODUCE

```
POST a sample declaration with valid target* properties using 3.9.0. 
```

##### EXPECTED RESULTS

```
Declaration(s) succeed -- presumably without installing additional components. In the alternative, the additional components install correctly and only once and there is a suitable pattern for dealing with this requirement.
```

##### ACTUAL RESULTS

See the follow [gist](https://gist.github.com/kreynoldsf5/08fadbe2a157b9b6a8570c58a19a5d9f).

**Comment by dstokesf5** [comment]
This issue has already been fixed for the upcoming 3.10 release. Thank you for reporting it.

**Comment by kaustriaf5** [comment]
Tracked internally as 880.

@kreynoldsf5,
Please see release [3.10](https://github.com/F5Networks/f5-appsvcs-extension/releases/tag/v3.10.0) for the fix.  Closing this issue.
